content/en/_index.html

```diff
@@ -38,62 +38,68 @@ <h4 class="section-head">What is Kubeflow?</h4>
   </div>
 </div>
 
+<div id="cncf">
+  <div class="contain">
+    <h4 class="section-head">We are a Cloud Native Computing Foundation project</h4>
```

Comment:
nit: Similar to other CNCF projects, I think we should link CNCF here 
ex. https://knative.dev/docs/ and https://kubernetes.io/


```suggestion
    <h4 class="section-head">We are a <a href="https://www.cncf.io/">Cloud Native Computing Foundation</a> project</h4>
```

Reply:
@annajung I think its 50/50, for example https://istio.io/ does not use it.

Personally, I am against links because they encourage users to click them, and in this case, that takes them of our website.